packages/odyssey/src/scss/abstracts/_mixins.scss

```diff
@@ -14,13 +14,40 @@
   position: absolute;
   width: 1px;
   height: 1px;
-  margin: -1px;
-  padding: 0;
+  margin-block: -1px;
+  margin-inline: -1px;
+  padding-block: 0;
+  padding-inline: 0;
   overflow: hidden;
   clip: rect(0 0 0 0);
   border: 0;
 }
 
+/* stylelint-disable liberty/use-logical-spec */
+
+@mixin border-radius( $all, $top-right: $all, $bottom-right: $all, $bottom-left: $all, $top-left: $all ) {
```

Comment:
Is there a way we enforce the use of this mixin beyond convention?

Reply:
Oh I guess we'd need to add `border-radius` to the disallow list...

Reply:
That's the current solve. If we want to create a custom rule, we could make the behavior more elaborate as well.